easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.0.0_fix-scp-path.patch

```diff
@@ -0,0 +1,23 @@
+fix path to stereo_chemical_props.txt when AlphaFold is installed with EasyBuild
+author: Kenneth Hoste (HPC-UGent)
+diff --git a/alphafold/common/residue_constants.py b/alphafold/common/residue_constants.py
+index 318b44a..fd840e3 100644
+--- a/alphafold/common/residue_constants.py
++++ b/alphafold/common/residue_constants.py
+@@ -19,6 +19,7 @@ import functools
+ from typing import Mapping, List, Tuple
+ 
+ import numpy as np
++import os
+ import tree
+ 
+ # Internal import (35fd).
+@@ -403,7 +404,7 @@ def load_stereo_chemical_props() -> Tuple[Mapping[str, List[Bond]],
+     residue_bond_angles: dict that maps resname --> list of BondAngle tuples
+   """
+   stereo_chemical_props_path = (
+-      'alphafold/common/stereo_chemical_props.txt')
++      os.path.join(os.getenv('EBROOTALPHAFOLD', 'alphafold/common'), 'stereo_chemical_props.txt'))
```

Comment:
This path does not exist (there is no `alphafold` directory in `$EBROOTALPHAFOLD`). We are putting the file in the `installdir`: https://github.com/easybuilders/easybuild-easyconfigs/pull/13744/files#diff-aadd0d22de19705201153354111a0b403bd18d8ded89c313ec684d909cbcb21aR148

Reply:
That's also what this line does, the path is correct (`stereo_chemical_props.txt` directly in `$EBROOTALPHAFOLD`) , we're just keeping the default here.

`alphafold/common` is just a fallback in case `$EBROOTALPHAFOLD` is not defined.

Reply:
Argh. I misread the brackets.